context/app/static/js/components/genes/SummaryBody/SummaryBody.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import SummaryPaper from 'js/shared-styles/sections/SectionPaper';
+import LabelledSectionText from 'js/shared-styles/sections/LabelledSectionText';
+
+function SummaryBody() {
+  return (
+    <SummaryPaper>
+      <LabelledSectionText label="Description" bottomSpacing={1}>
+        Our best estimates show there are over 7 billion people on the planet and 300 billion stars in the Milky Way
```

Comment:
Add a comment to make it more clear this is temporary?

Reply:
Done